test/functional/services/dashboard/add_panel.js

```diff
@@ -170,7 +182,7 @@ export function DashboardAddPanelProvider({ getService, getPageObjects }) {
       log.debug(`DashboardAddPanel.addVisualization(${vizName})`);
       await this.ensureAddPanelIsShowing();
       // workaround for timing issue with slideout animation
-      await PageObjects.common.sleep(500);
+      // await PageObjects.common.sleep(500);
```

Comment:
I think this line and the line above should just be removed

Reply:
Yes, I'll remove that.  I commented the sleep out in this PR hoping and expecting we wouldn't need it.  I'm going to run this against Kibana in Cloud just to make sure it works there too.